modules/stats.php

```diff
@@ -181,8 +181,10 @@ function stats_footer() {
 function stats_get_options() {
 	$options = get_option( 'stats_options' );
 
-	if ( !isset( $options['version'] ) || $options['version'] < STATS_VERSION )
+	if ( ( ! isset( $options['version'] ) || $options['version'] < STATS_VERSION ) &&
+		( is_admin() || ( defined( 'DOING_CRON' ) && DOING_CRON ) ) ) {
```

Comment:
Throughout `stats.php`, we're assuming the output of will be an array of `stats_get_options` options. If someone installed Jetpack via CLI, I don't think this would ever fire to get the default options, so most of the code would at least throw notices about the undefined indexes until a cron run or someone visited wp-admin.

Could spin out a "stats_default_options" that returns the default array, `if ( false === $options ) { return stats_default_options()}` and use that same function in `stats_upgrade_options` to keep things DRY.

Thoughts?